File: Source/OregonTilth.Web/src/app/shared/models/forms/field-input-cost/field-input-cost-create-dto.ts
export class FieldInputCostCreateDto {
    WorkbookID: number;
    FieldInputCostName: string;
    FieldUnitTypeID: number;
    CostPerFieldUnit: number;
    Notes: string;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: OregonTilth.Models/DataTransferObjects/Workbook/Forms/FieldInputCost/FieldInputCostCreateDto.cs
using System.ComponentModel.DataAnnotations;

namespace OregonTilth.Models.DataTransferObjects
{
    public class FieldInputCostCreateDto
    {
        [Required] public int WorkbookID { get; set; }
        [Required] public int FieldUnitTypeID { get; set; }
        [Required] public string FieldInputCostName { get; set; }
        [Required] public decimal CostPerFieldUnit { get; set; }
        public string Notes { get; set; }

    }
}